Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/ReviewAddOwnerTxViewController.swift

```diff
@@ -74,8 +76,8 @@ class ReviewAddOwnerTxViewController: ReviewSafeTransactionViewController {
         let (name, _) = NamingPolicy.name(for: owner,
                                                     info: nil,
                                                     chainId: safe.chain!.id!)
-
-        cell.set(owner: AddressInfo(address: owner, name: name), action: .addingOwner, prefix: safe.chain?.shortName)
+        let newName = newAddressName ?? name
```

Comment:
I think this should be the opposite
`let newName = name ?? newAddressName `

This way owner keys table will have more priority than new name which might be address book somehow 
@DmitryBespalov is this aligned with our requiremet ?

Reply:
`newAddressName` is the name just entered by the user. IMHO that should have precedence of any `name` form the database. 

Only if `newAddressName` is nill, then the fallback `name` is used.

We could make this more efficient by not calling `NamingPolicy.name()` id newAddressName is not nil.

Reply:
```
        if let newName = newAddressName {
            cell.set(owner: AddressInfo(address: owner, name: newName), action: .addingOwner, prefix: safe.chain?.shortName)
        } else {
            let (name, _) = NamingPolicy.name(for: owner, info: nil, chainId: safe.chain!.id!)
            cell.set(owner: AddressInfo(address: owner, name: name), action: .addingOwner, prefix: safe.chain?.shortName)
        }
```

Like this. Is it possible to have only one call `cell.set()`?